.github/workflows/publish.yml

```diff
@@ -0,0 +1,74 @@
+name: publish
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+
+  build-test:
+    name: Build-Test
+    strategy:
+      matrix:
+        go-version: [1.15.x]
+        platform: [ubuntu-latest]
+
+    runs-on: ${{ matrix.platform }}
+
+    steps:
+      - name: Set up Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+        id: go
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: build
+        run: make build
+      - name: Test
+        run: make test
+
+  publish-derek:
+    runs-on: ubuntu-latest
+    needs: build-test
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Get Repo Owner
+        id: get_repo_owner
+        run: echo ::set-output name=repo_owner::$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]')
+      - name: Get TAG
+        id: get_tag
+        run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+      - name: Login to GHCR Registry
+        uses: docker/login-action@v1
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          registry: ghcr.io
+      - name: Login to Quay.io
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.QUAY_USERNAME }}
```

Comment:
I would expected this to be `ghcr.io` rather than quay.io, I'm not sure if that was here before in the Travis config, but we've moved away from using Quay as a community.

Otherwise it looks good to me.

Reply:
Travis had both docker hub and quay.io.
Do you want to push just to GHCR?

Reply:
That's right, most of the projects just push to GHCR now, to avoid the rate limits of the Docker Hub.

Reply:
After that I'll get a release build done and can update the production server for managed Derek.

Reply:
ok, will remove quay one sec

Reply:
thanks for your review and feedback
PTAL